data_structures/src/utxo_pool.rs

```diff
@@ -206,16 +216,47 @@ impl<'a> OutputsCollection for NodeUtxos<'a> {
 }
 
 /// Strategy to sort our own unspent outputs pool
-#[derive(Clone, Copy, Debug, Serialize, Deserialize, Eq, PartialEq, Hash)]
+#[derive(Clone, Debug, Eq, PartialEq, Hash)]
 pub enum UtxoSelectionStrategy {
-    Random,
-    BigFirst,
-    SmallFirst,
+    Random { from: Option<PublicKeyHash> },
+    BigFirst { from: Option<PublicKeyHash> },
+    SmallFirst { from: Option<PublicKeyHash> },
```

Comment:
```suggestion
    #[serde(rename = "random")]
    Random { from: Option<PublicKeyHash> },
    #[serde(rename = "big_first")]
    BigFirst { from: Option<PublicKeyHash> },
    #[serde(rename = "small_first")]
    SmallFirst { from: Option<PublicKeyHash> },
```

Reply:
Did this for `UtxoSelectionStrategyName`, which is where serialization is implemented. Also, using the old names is still allowed because of backward-compatibility in the node JSON-RPC API.